src/main/features/response/views/when-will-you-pay.njk

```diff
@@ -0,0 +1,48 @@
+{% extends "layout.njk" %}
+
+{% set title = 'Money claim' %}
+
+{% from "form.njk" import saveAndContinueButton, csrfProtection, radioGroup, radioGroupWithHintPanel, errorSummary %}
+
+{% block content %}
+  <div class="column-two-thirds">
+
+    <h1 class="heading-large">
+      {{ t("When will you pay?") }}
+    </h1>
+
+    <form method="post"
+          class="analytics-submit-event-trigger"
+          data-event-action="Defendant: Select payment option"
+          data-event-label-from="option"
+          novalidate>
+      {{ csrfProtection(csrf) }}
+
+      {{
+      radioGroupWithHintPanel(
+        label = 'I would like the defendant to be ordered to pay:',
+        form = form,
+        name = 'option',
+        options = [
+          {
+            label: 'The full amount on set date',
+            value: 'FULL_BY_SPECIFIED_DATE',
+            checked: (form.valueFor('option').value === 'FULL_BY_SPECIFIED_DATE')
+          },
+          {
+            label: 'By instalments',
+            value: 'INSTALMENTS',
+            checked: (form.valueFor('option').value === 'INSTALMENTS'),
+            hint: t('You will need to give details of your finances to {{ claimantFullName }} to explain why.',
```

Comment:
Please update macro so that it does the translation and pass free text here.

Reply:
I will remove the translation here but I think in the future we need to think about refactoring the whole form.njk

Reply:
Actually this breaks the route test when removing the translation and doesn't render template correctly

Reply:
My bad, translation needs personalisation. There is no way of doing this in macro :(